extensions/android-paging3/android-test/src/test/java/app/cash/sqldelight/paging3/OffsetQueryPagingSourceTest.kt

```diff
@@ -629,14 +629,40 @@ class OffsetQueryPagingSourceTest {
     assertTrue(pagingSource.jumpingSupported)
   }
 
-  private fun query(limit: Int, offset: Int) = object : Query<TestItem>(
+  @Test
+  fun load_initialEmptyLoad_QueryPagingSourceLong() = runTest {
```

Comment:
Just wanted a test to ensure that the constructor works as expected. This test is a copy of `load_initialEmptyLoad`, just using `Long` instead of `Int`.